src/components/shared/Scheduler.vue

```diff
@@ -289,7 +289,7 @@ Licensed under the Elastic License 2.0. */
         :min-date="new Date(studyStore.study.plannedStart as string)"
         :max-date="new Date(studyStore.study.plannedEnd as string)"
         :manual-input="false"
-        :placeholder="'hh:mm'"
+        placeholder="hh:mm"
```

Comment:
Warum ist das nicht getemplated? also ge-`$t()`?

Reply:
Because the primevue calendar component does not differ between different locale hours/minute formats. We only have the prop "time-only" to specify a time input and it's always shown as hh:mm, therefore the placeholder has hh:mm hardcoded.